src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,118 @@
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[] | null;
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      <ul>
+        {todos?.map(todo => (
+          <li key={todo.id}>
+            <TodoItem todo={todo} />
+          </li>
+        ))}
+      </ul>
+
+      {/* <div data-cy="Todo" className="todo">
```

Comment:
don't leave comments
if you need them for the next steps you can save them in another branch (it is also always available in the master branch).